specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -539,6 +539,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/hostingEnvironments/{name}/inboundnetworkdependenciesendpoints": {
```

Comment:
inboundnetworkdependenciesendpoints feels quite long-winded and it's not obvious to me what this actually is. Let's have a discussion to go over some of the naming.

Reply:
Unfortunately, as I mentioned before, this API is in production and the signature can't be changed in this version. I'll work on options in the API name.  Does dropping "Endpoints" make more sense? Initially, the api was a list of <IP, Port> but the requirements changed. InboundNetworkDependencies and OutboundNetworkDependencies?

Reply:
Understood. Would you please lowerCamelCase the type name? `inboundNetworkDependenciesEndpoints`